recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -51,10 +51,10 @@
    "id": "a9c72d26",
    "metadata": {},
    "source": [
-    "In order to run this notebook, one needs to have installed Gammapy as well as PINT (see documentation above) in the same environment. We recommend to first install Gammapy and then install PINT using your prefered package manager.\n",
+    "To run this notebook, you must have Gammapy and PINT (see documentation above) installed in the same environment. We recommend installing Gammapy first and then installing PINT using your preferred package manager.\n",
     "\n",
     "\n",
-    "`$ conda env create -n gammapy-pint -f gammapy-1.0-environment.yml`\n",
+    "`$ conda env create -n gammapy-pint -f gammapy-pint-environment.yml`\n",
```

Comment:
What is this `gammapy-pint-environment.yml` ?

Reply:
I don't really know. I think that is was the original env file that was presented to the user. But then we also need this `env.yml` for the CI. In the end they are the same so I could make `gammapy-pint-environment.yml` a simlink to env.